src/TraceEvent/Computers/StartStopActivityComputer.cs

```diff
@@ -708,7 +709,7 @@ internal static unsafe string CreateActivityPathString(Guid guid)
             {
                 uint nibble = (uint)(*bytePtr >> 4);
                 bool secondNibble = false;              // are we reading the second nibble (low order bits) of the byte.
-                NextNibble:
+            NextNibble:
```

Comment:
Nit: Can you remove this whitespace diff please?

Reply:
Sure.